code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -96,8 +96,14 @@
 			return
 		if(..() == 1)
 			playsound(src.loc, "swing_hit", 50, 1, -1)
-			if(target_zone == "r_leg" || target_zone == "l_leg")
-				target.Weaken(5) //nerfed, because yes.
+			if (target_zone == "r_leg" || target_zone == "l_leg")
+				if(ishuman(target))
+					var/mob/living/carbon/human/T = target
+					var/armor = T.run_armor_check(T, "melee")
+					if(armor < 100)
+						T.Weaken(5) //nerfed, because yes.
+			else

```

Comment:
This is logically invalid. IF targetzone is r_leg or l_leg (...) ELSE (weaken target.)

The else here should be on related to the `ishuman(target)` if. IF ishuman (...) ELSE (weaken target.)